src/module.cpp

```diff
@@ -479,7 +479,8 @@ std::unordered_map<instruction_ref, std::string> module::print(
         }
         else
         {
-            var_name = this->name() + ":@" + std::to_string(count);
+            var_name = "@" + std::to_string(count);
+            var_name = this->name().empty() ? var_name : (this->name() + ":" + var_name);
```

Comment:
I dont think this changed should be in this PR.

Reply:
OK, I will move to the other PR I have.